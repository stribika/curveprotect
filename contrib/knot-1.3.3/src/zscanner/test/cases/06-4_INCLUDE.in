$ORIGIN .
$TTL	1

; KO
$INCLUDE /home/ondrej/Projects/knot/src/zscanner/test/cases/zscanner_tests/file-doesnt-exist		; File doesn't exist
