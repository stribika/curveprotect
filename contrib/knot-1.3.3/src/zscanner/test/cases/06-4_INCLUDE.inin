$ORIGIN .
$TTL	1

; KO
$INCLUDE @abs_builddir@/zscanner_tests/file-doesnt-exist		; File doesn't exist
