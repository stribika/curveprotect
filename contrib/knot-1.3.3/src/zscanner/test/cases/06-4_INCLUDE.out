ERROR=ZSCANNER_EUNOPENED_INCLUDE
------
